### PR TITLE
fix(AV-1642): Add link support for guide menu headers

### PR DESCRIPTION
**Pull request by beazil**

### CODE REVIEW

File: modules/avoindata-theme/avoindata.theme [hunk 1]
```diff
@@ -18,6 +18,23 @@ function avoindata_preprocess_region(array &$variables) {
   $variables['language'] = \Drupal::languageManager()->getCurrentLanguage()->getId();
 }
 
+function avoindata_preprocess_block(&$variables) {
+  $variables['link_map'] = [
+    'aboutopendatamenuen' => '/en/about-open-data',
+    'aboutopendatamenufi' => '/fi/tietoa-avoimesta-datasta',
+    'aboutopendatamenusv' => '/sv/api-principer',
```
Comment (beazil) [inline modules/avoindata-theme/avoindata.theme:25]: Hmm, I guess that this should be something else. @Zharktas I believe we had somewhere list about these url:s, I could maybe fix it in block definition also 🤔

Comment (Zharktas) [reply modules/avoindata-theme/avoindata.theme:25]: Yeah it should be /information-om-oppna-data/, the list is here https://wiki.dvv.fi/display/AV/Rakenne

Comment (Zharktas) [reply modules/avoindata-theme/avoindata.theme:25]: Its my copy paste error in the block definition though :D

Comment (beazil) [reply modules/avoindata-theme/avoindata.theme]: Thanks, I missed that also in the first place

**Review by Zharktas — APPROVED** [review]

**Pull request merged**